projects/copper-coreengine/src/main/java/org/copperengine/core/persistent/AbstractSqlDialect.java

```diff
@@ -73,11 +74,12 @@
     protected StmtStatistic selectQueueSizeStmtStatistic;
 
     public AbstractSqlDialect() {
-        this(false, false);
+        this(false, false, true);
     }
 
-    public AbstractSqlDialect(final boolean supportsMultipleEngines, final boolean defaultMultiEngineMode) {
```

Comment:
This constructor is public API. Changing its signature would be a breaking change and it would require a major version change. My suggestion is to keep the old constructor (and maybe mark it as deprecated) and additionally provide your new one:
```
@Deprecated
public AbstractSqlDialect(final boolean supportsMultipleEngines, final boolean defaultMultiEngineMode) {
    this(supportsMultipleEngines, defaultMultiEngineMode, true);
}

 public AbstractSqlDialect(final boolean supportsMultipleEngines, final boolean defaultMultiEngineMode, boolean supportsClob) {
    // your code here
}
```